homeassistant/components/zha/core/channels/lighting.py

```diff
@@ -45,6 +45,7 @@ class ColorChannel(ZigbeeChannel):
         "color_capabilities": True,
         "color_loop_active": False,
         "start_up_color_temperature": True,
+        "options": True,
```

Comment:
Technically this isn't really needed at the moment, as the ``options`` parameter needs to be set manually anyway (for now) and the ``options`` property defaults to ``0`` if there isn't a cached attribute.

In the future, when a configuration entity is added for this attribute (for example), it would be nice if that attribute was already read/initialized (so the configuration entity is created appropriately).